site/content/blogs/release-v120.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'AWS Copilot v1.20: To Environment Manifests and Beyond!'
+twitter_title: 'AWS Copilot v1.20'
+image: 'https://user-images.githubusercontent.com/879348/179278910-1e1ae7e7-cb57-46ff-a11c-07919f485c79.png'
+image_alt: 'Environment manifests'
+image_width: '1106'
+image_height: '851'
+---
+
+# AWS Copilot v1.20: To Environment Manifests and Beyond!
+
+The AWS Copilot core team is announcing the Copilot v1.20 release.  
+Special thanks to [@gautam-nutalapati](https://github.com/gautam-nutalapati), [@codekitchen](https://github.com/codekitchen), [@kangere](https://github.com/kangere/) who contributed to this release.
+Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and has over 300 people online and over 2.3k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.20 brings several new features and improvements:
+
+* **Environment manifests**: You can now create and update environments with a [manifest file](../docs/manifest/environment.en.md) providing all the benefits of infrastructure as code to environments. 
+   [See detailed walkthrough](#environment-manifest) for how to migrate your existing environments.
+
+
+???+ note "What’s AWS Copilot?"
+
+    The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.  
+    From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+    At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code.
+    Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro service architectures,
+    enabling you to focus on developing your application, instead of writing deployment scripts.
+    
+    See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## Environment Manifest
+
+Prior to v1.20, clients could not update their environments with additional configuration. For example, if your environment wasn't associated
+with a domain, users would need to run `env init --name copy --import-cert-arns` to create a new environment with certificates and then tear down the old one.
+Starting with this release, users can modify their environments using the [manifest](../docs/manifest/environment.en.md) without having
+to recreate them.  
+Moving forward new environment configurations will only be surfaced via the `manifest.yml` file instead of flags in the `env init` command.
```

Comment:
We have workload manifest from the very beginning and we still have flags for `svc init`. I think it is just we don't necessarily expose all config by adding flags 🤔